rally-domain/src/integration-test/groovy/yakworks/rally/orgs/ContactTests.groovy

```diff
@@ -112,24 +112,48 @@ class ContactTests extends Specification implements DomainIntTest {
         ex.code == DataProblemCodes.ReferenceKey.code
     }
 
+    void "test set contact as primary contact"() {
+        setup:
+        Org contactOrg = Org.get(50)
+        Contact contact = contactOrg.contact
+
+        expect:
+        contact != null
+        contact.isPrimary
+
+        when:
+        contactOrg.contact = null
+        contactOrg.persist()
+
+        then:
+        contact.isPrimary == false
+
+
+        when:
+        contact.repo.update(id:contact.id, isPrimary:true)
+
+        then:
+        contact.isPrimary == true
+    }
+
     void "test create Contact with org lookup by orgSource"() {
         setup:
         Org org = Org.of("foo", "bar", OrgType.Customer)
         Org.repo.createSource(org)
         org.persist(flush: true)
 
-        Map params = [firstName:'Peter', email:'abc@walmart.com']
+        Map params = [firstName:'Peter', email:'abc@walmart.com', isPrimary:"true"]
```

Comment:
why is true a string? fix or comment, I cant thing of a case where we would do this